go.mod

```diff
@@ -39,6 +36,7 @@ require (
 	go.etcd.io/bbolt v1.3.5
 	go.etcd.io/etcd/api/v3 v3.0.0-00010101000000-000000000000
 	go.etcd.io/etcd/pkg/v3 v3.0.0-00010101000000-000000000000
+	go.etcd.io/etcd/raft/v3 v3.0.0-00010101000000-000000000000
```

Comment:
What's the versioning story here? Multi-module repos behave in unexpected ways for consumers if the version here does not point to the code at HEAD. Making a tagged release will require:

1. tag the raft module version
3. Update main module to reference the tagged version of the raft module
4. Commit and push the main module
5. Tag the main module

Reply:
I assume we will do this in 2 step process. 

During the release we will: 
  1. replace across all modules  v3.0.0-00010101000000-000000000000 with v3.5.2 in a single commit
        (so both raft & main module (and others) ).
  2. tag the repo as v3.5.2